### DISCUSSION TITLE
GarminDB and timezones

### DISCUSSION THREAD

**Discussion by marcin-przywoski** [tcgoetz/GarminDB#215] [Q&A] [answered]
Hello everyone!

I have a question - how does GarminDB handle timezone changes. This is important for me as I'm from Poland where we have winter time and summer time which changes in March and October. I have a script that exports the data into XML file which I can then import into Apple Health (as Garmin Connect on iOS does not import respiratory rate and oxygen into Apple Health) but the problem that I have is how to handle the timezones. I'm not sure how Garmin handles that and if it's handled in anyway during creation of the DB. Does rebuilding the DB work for that case by re-adjusting the timestamp data based on what is in the .fit files and the current timezone?

**Comment by tcgoetz** [comment, marked as answer]
The timezone adjustments are done by Garmin Connect and is in data downloaded from GC.

**Reply by marcin-przywoski** [reply]
Do we have a way in which we can check the timezone for the current entry? I can see in the db that there is a timezone entry but I checked the version of the db that was made during summer time and the timezone was the same. Does that mean that garmin has it fixed?

**Reply by marcin-przywoski** [reply]
nevermind - checked how it's set in GC, looks like it's always in GMT+1
